tests/auto/rt_auto.py

```diff
@@ -245,28 +260,46 @@ def run_function(self):
     def move_rt_logs(self):
         ''' This is the callback function associated with the "RT" command '''
         logger = logging.getLogger('JOB/MOVE_RT_LOGS')
-        rt_log = f'tests/RegressionTests_{self.machine["name"]}.log'
+        rt_log = f'tests/RegressionTests_{self.machine["name"]}.intel.log'
         filepath = f'{self.pr_repo_loc}/{rt_log}'
-        rm_filepath = '/'.join((self.pr_repo_loc.split('/'))[:-1])
         if os.path.exists(filepath):
             move_rt_commands = [
+                [f'git pull --ff-only origin {self.branch}', self.pr_repo_loc],
                 [f'git add {rt_log}', self.pr_repo_loc],
-                [f'git commit -m "Auto: Added Updated RT Log file: {rt_log}"', self.pr_repo_loc],
-                [f'git pull --no-edit origin {self.branch}', self.pr_repo_loc],
+                [f'git commit -m "Auto: Add RT Log file: {rt_log} skip-ci"', self.pr_repo_loc],
                 ['sleep 10', self.pr_repo_loc],
                 [f'git push origin {self.branch}', self.pr_repo_loc]
             ]
             self.run_commands(move_rt_commands)
 
         else:
-            logger.critical('Could not find RT log')
-            raise FileNotFoundError('Could not find RT log')
+            logger.critical('Could not find Intel RT log')
+            raise FileNotFoundError('Could not find Intel RT log')
+
+    def move_grt_logs(self):
+        ''' This is the callback function associated with the "GRT" command '''
+        logger = logging.getLogger('JOB/MOVE_GRT_LOGS')
+        rt_log = f'tests/RegressionTests_{self.machine["name"]}.gnu.log'
+        filepath = f'{self.pr_repo_loc}/{rt_log}'
+        if os.path.exists(filepath):
+            move_grt_commands = [
+                [f'git add {rt_log}', self.pr_repo_loc],
+                [f'git commit -m "Auto: Added Updated RT Log file: {rt_log} skip-ci"', self.pr_repo_loc],
+                [f'git pull --no-edit origin {self.branch}', self.pr_repo_loc],
+                ['sleep 10', self.pr_repo_loc],
+                [f'git push origin {self.branch}', self.pr_repo_loc]
+            ]
+            self.run_commands(move_grt_commands)
+
+        else:
+            logger.critical('Could not find Gnu RT log')
+            raise FileNotFoundError('Could not find Gnu RT log')
```

Comment:
`move_rt_logs` and `move_grt_logs` are virtually identical. Consider using one function and pass the compiler name as a parameter.

Reply:
i completely agree and told myself someone would mention it. My brain is pretty fried and will merge those two functions next week at some point.

Reply:
Done in newest commit.